client/src/main/java/io/apicurio/registry/rest/client/RegistryClient.java

```diff
@@ -58,9 +58,9 @@
 
 	void testUpdateArtifact(String groupId, String artifactId, InputStream data);
 
-	InputStream getArtifactVersion(String groupId, String artifactId, String version);
+	InputStream getArtifactVersion(String groupId, String artifactId, Integer version);
 
-	VersionMetaData getArtifactVersionMetaData(String groupId, String artifactId, String version);
+	VersionMetaData getArtifactVersionMetaData(String groupId, String artifactId, Integer version);
```

Comment:
String is actually correct here - I'm prepping the API to support user-provided Semver formatted versions.

Reply:
reverted